server/game/core/card/Card.ts

```diff
@@ -38,6 +38,9 @@ export class Card extends OngoingEffectSource {
     public static implemented = false;
 
     public readonly aspects: Aspect[] = [];
+    public readonly smuggleAspects?: Aspect[];
+    public readonly smuggleCost?: number;
+    public readonly additionalSmuggleCosts?: boolean;
```

Comment:
Just checking, is this temporary? I was expecting that the smuggle details would be wrapped up in the keyword instance, since that's how it's done for raid and restore and there might be multiple instance of smuggle present on one card so it can't necessarily be represented like this.